client/src/routes/widgetModule/WidgetModule.tsx

```diff
@@ -95,6 +125,188 @@ export const WidgetModule: React.FC<Props> = observer(({ domainName }) => {
     setFormFields({ ...formFields, widgetValue: '' })
   }
 
+  const createAlias = async (alias: string, forward: string) => {
+    setProcessStatus({
+      type: ProcessStatusTypes.PROGRESS,
+      render: (
+        <BaseText>
+          {walletStore.isMetamaskAvailable
+            ? 'Waiting for a transaction to be signed'
+            : 'Sign transaction on mobile device'}
+        </BaseText>
+      ),
+    })
+
+    try {
+      const numAlias = await rootStore.easClient.getNumAlias(domainName)
+      const maxAlias = await rootStore.easClient.maxNumAlias()
+      const publicAliases = await rootStore.easClient.getPublicAliases(
+        domainName
+      )
+      console.log('### maxNum', numAlias)
+      console.log('### numAlias', numAlias)
+      console.log('### publicAliases', publicAliases)
+
+      if (numAlias >= maxAlias) {
+        setProcessStatus({
+          type: ProcessStatusTypes.PROGRESS,
+          render: (
+            <BaseText>
+              {walletStore.isMetamaskAvailable
+                ? 'Waiting for a transaction to be signed'
+                : 'Sign transaction on mobile device'}
+            </BaseText>
+          ),
+        })
+
+        const removingAlias = publicAliases[0]
```

Comment:
There could also be non-public alias unless enforced by the app. Need to check

Reply:
currently we allow to create public alias only, and currently i have no information about non-public alias feature, so we should check this

Reply:
okay, maybe later